src/storage/es_posix.c

```diff
@@ -132,7 +132,7 @@ es_make_dirs (const char *dirname1, const char *dirname2)
       assert (false);
       return ER_ES_INVALID_PATH;
     }
-  ret = mkdir (dirbuf, 0755);
+  ret = mkdir (dirbuf, 0744);
```

Comment:
The 'members of a group' and 'others' have read permission. I think that one of the reasons for this change is to protect our files created by the server regardless of the value of umask.

Reply:
Ignore this comment. I forgot to set the umask(S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH) on log_initialize_internal() for a moment.